### DISCUSSION TITLE
Augment Face

### DISCUSSION THREAD

**Discussion by Amitdygon** [google/filament#5992] [General]
Hello All

Is there any way that i can use this Filament for Augment Face or set 3D model on face.

Thanks

**Comment by developerGM** [comment]
@Amitdygon 
Did you solved?

**Reply by Amitdygon** [reply]
Yes i found a alternative  way https://github.com/Amitdygon/sceneform-android-sdk here this libarary have filament and google ar core or augmented  face ar all in one. Please try it

**Reply by developerGM** [reply]
Thanks. I found this
https://github.com/SceneView/sceneform-android
but I will try also your suggestion

**Reply by Amitdygon** [reply]
This is the same library that i suggested you.
